app/src/main/java/ch/epfl/sdp/peakar/points/ComputePOIPoints.java

```diff
@@ -64,6 +85,16 @@ public ComputePOIPoints(Context context){
      * @param userPoint user location
      */
     private static void getPOIs(UserPoint userPoint){
+        // clear the old points
+        POIs.clear();
+
+        // first check that if offline mode is active
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(ctx);
+        boolean offlineModeValue = prefs.getBoolean(ctx.getResources().getString(R.string.offline_mode_key), false);
```

Comment:
Should the offline POIs be used instead of the cached POIs?

Reply:
I'll investigate it as my task for next week together with using the downloaded content as a cache